serverless.yml

```diff
@@ -0,0 +1,50 @@
+service: poprox-default-recommender
+frameworkVersion: "3"
+
+provider:
+  name: aws
+  runtime: python3.11
+  versionFunctions: false
+  region: ${opt:region, "us-east-1"}
+  stage: ${opt:stage, "prod"}
+  memorySize: 512
+  timeout: 900
+
+  iam:
+    role:
+      # Add statements to the IAM role to give permissions to Lambda functions
+      statements:
+        - Effect: Allow
+          Action:
+            - "logs:*"
+          Resource: "*"
+
+plugins:
+  - serverless-offline
+  - serverless-python-requirements
+custom:
+  pythonRequirements:
+    dockerizePip: true
+    useDownloadCache: false
+    useStaticCache: false
+    slim: true
+    strip: false
+    slimPatterns:
+      - "**/*.csv"
+
+package:
+  patterns:
+    - "!venv/**"
+    - "!models/**"
+    - "!node_modules/**"
+    - "!tests/**"
+    - "**.py"
+    - "**.html"
+
+functions:
```

Comment:
This is where we link incoming request URLs/paths to specific Python functions that should run to process the request. `handler` is just the import path of the function, and `httpAPI: path` is where we set the URL. It could be something like `/generate` to be explicit, but I think there's only one URL and one function needed so maybe `/` is okay?